frontend/apps/reader/modules/readerflipping.lua

```diff
@@ -40,18 +40,13 @@ function ReaderFlipping:init()
         alpha = true,
     }
     self[1] = LeftContainer:new{
-        dimen = Geom:new{w = Screen:getWidth(), h = self.flipping_widget:getSize().h},
+        dimen = Geom:new{w = self.flipping_widget:getSize().w, h = self.flipping_widget:getSize().h},
```

Comment:
I guess this LeftContainer is a no-op/null-container then :) and we could have `self[1] = self.flipping_widget`
But ok for keeping it as-is and not change anything else that works.

Reply:
Huh, yeah, looks like it.

Reply:
Oh, wait, it might be useful to handle mirroring in RTL mode?

Reply:
Oh, possibly indeed :)
So, may be your change of reducing its width from the screen width to the icon size is wrong ?

Reply:
Oh, dammit, I was just thinking this might at least break the *refreshes* in RTL, but you're right, it probably breaks positioning, too :/.

Welp, I desperately need a nap, so, more on that later ;).

Reply:
There, fixed it. And did something slightly more elegant for the refresh region (i.e., we can just use the IconWidget's dimen object).